docker/compose.server.yml

```diff
@@ -63,12 +63,11 @@ services:
       - "traefik.http.routers.u2.entryPoints=web"
       - "traefik.http.routers.u2.rule=Host(`${SERVER_DNS}`)&&PathPrefix(`/${username2}`)"
       - "traefik.http.routers.u2.middlewares=traefik-forward-auth"
-  
-  
+
   traefik-forward-auth:
     image: thomseddon/traefik-forward-auth:latest
     volumes:
-      - ${DTAAS_DIR}/docker/auth/conf:/conf
+      - ${OAUTH_CONF_FILEPATH}:/conf
```

Comment:
given that `conf.auth` is only useful for compose.server.yml, we might as well hardcode it here.